front/src/Stores/MenuStore.ts

```diff
@@ -200,7 +200,7 @@ inviteUserActivated.subscribe((value) => {
         (menu) => (menu as TranslatedMenu).key === SubMenusInterface.invite
     );
     if (value && indexInviteMenu === -1) {
-        valuesSubMenusStore.push(inviteMenu);
+        valuesSubMenusStore.splice(3, 0, inviteMenu);
```

Comment:
It would be cleaner to put this subscription inside the `function createSubMenusStore()`

You could use the update function there to update the store?